### PR TITLE
Update CHANGELOG.md for 2.2.0

### PR DESCRIPTION
**Pull request by jforissier**
Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>

### CODE REVIEW

**Comment by jbech-linaro** [comment]
Looking good, let's wait for the RB's and TB's until the end of the week.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,107 @@
...
+* rcar-h3: TBD
+* rpi3: TBD
+* sunxi: TBD
+* ti-dra7xx: TBD
+* vexpress-fvp: TBD
+* vexpress-juno: TBD
+* vexpress-qemu: TBD
+* vexpress-qemu_armv8a: TBD
+* vexpress-qemu_virt: TBD
+* zynqmp-zcu102: TBD
```
Comment (sorenb-xlnx) [inline CHANGELOG.md]: Standard tests pass on zynqmp.

Comment (jforissier) [inline CHANGELOG.md]: Thanks @sorenb-xlnx, change log updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,112 @@
...
+## Tested on
+
+In the list below, _standard_ means that the `xtest` program passed with
+its default configuration, while _extended_ means it was run successfully
+with the additional GlobalPlatform™ TEE Initial Configuration Test Suite
+v1.1.0.4.
+
+<!-- ${PLATFORM}-${PLATFORM_FLAVOR}, ordered alphabetically -->
+* d02: TBD
+* hikey: TBD
```
Comment (jforissier) [inline CHANGELOG.md]: Extended tests pass on HiKey. More details: https://docs.google.com/document/d/1TUz1eAvhr13twZERD2H9W89KLX2ztEOniBF63SNE964/edit?usp=sharing

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,112 @@
...
+
+## Tested on
+
+In the list below, _standard_ means that the `xtest` program passed with
+its default configuration, while _extended_ means it was run successfully
+with the additional GlobalPlatform™ TEE Initial Configuration Test Suite
+v1.1.0.4.
+
+<!-- ${PLATFORM}-${PLATFORM_FLAVOR}, ordered alphabetically -->
+* d02: TBD
```
Comment (jforissier) [inline CHANGELOG.md]: Extended tests pass on HiKey.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,112 @@
...
+<!-- ${PLATFORM}-${PLATFORM_FLAVOR}, ordered alphabetically -->
+* d02: extended
+* hikey: extended
+* imx-mx6qsabrelite: TBD
+* imx-mx6qsabresd: TBD
+* imx-mx6ulevk: TBD
+* ls-ls1021atwr: TBD
+* mediatek-mt8173: TBD
+* rcar-h3: TBD
+* rpi3: TBD
```
Comment (jbech-linaro) [inline CHANGELOG.md]: Just tested and RPi3 seems to be working. To help you a bit I've generated a stable manifest file here. Do you want me to submit that as a PR to build.git?

```xml
<?xml version="1.0" encoding="UTF-8"?>
<manifest>
  <remote fetch="https://github.com/mirror" name="busybox_mirror"/>
  <remote fetch="https://github.com/linaro-swg" name="linaro-swg"/>
  <remote fetch="https://github.com/OP-TEE" name="optee"/>
  
  <project name="arm-trusted-firmware.git" path="arm-trusted-firmware" remote="linaro-swg" revision="ec4262bbeba3c7db609fb2aa77a4ddc6deeeab5f" upstream="rpi3_initial_drop"/>
  <project name="build.git" path="build" remote="optee" revision="c14e9526456622614dd4b79cdbf0a583aabd23e0" upstream="master">
    <linkfile dest="build/Makefile" src="rpi3.mk"/>
    <linkfile dest="build/pi3.cfg" src="rpi3/debugger/pi3.cfg"/>
    <linkfile dest="build/gdb" src="../toolchains/aarch64/bin/aarch64-linux-gnu-gdb"/>
  </project>
  <project name="busybox.git" path="busybox" remote="busybox_mirror" revision="dbf5a6da6a4295ce26edd1ce34fde567d19afa02"/>
  <project name="gen_rootfs.git" path="gen_rootfs" remote="linaro-swg" revision="7618525ca98116a75a8524a8a4d5b6cd8f9e2cb6" upstream="master"/>
  <project name="hello_world.git" path="hello_world" remote="linaro-swg" revision="4725dcc0e02c9e4f131e0d65db6617dfc782f55a" upstream="master"/>
  <project name="linux.git" path="linux" remote="linaro-swg" revision="cc225a78910c37d78f8a00c80dcbf59ef7762884" upstream="electron752-rpi3-optee"/>
  <project name="optee_client.git" path="optee_client" remote="optee" revision="658ae538f76a2624b7f9c40539a600d281d872b4" upstream="master"/>
  <project name="optee_os.git" path="optee_os" remote="optee" revision="287359f44187396c3e12d1bab257ce5918a58cb0" upstream="master"/>
  <project name="optee_test.git" path="optee_test" remote="optee" revision="a00eeb15ddfaa5b7d2cb340faeb36748d5aabffb" upstream="master"/>
  <project name="u-boot.git" path="u-boot" remote="linaro-swg" revision="10577db76e77a513180388df1f176a6642143c84" upstream="rpi3_initial_drop"/>
</manifest>
```

Comment (jbech-linaro) [inline CHANGELOG.md]: Also, adding a `rpi3_stable.xml` and rename  `rpi3_expermental.xml -> rpi3.xml`, will require changes to `README.md` in `optee_os.git`.

Comment (jbech-linaro) [inline CHANGELOG.md]: For manifest: https://github.com/OP-TEE/manifest/pull/37 has been created.

Comment (jbech-linaro) [inline CHANGELOG.md]: And for changes in OP-TEE OS: https://github.com/OP-TEE/optee_os/pull/1088 has been created.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,112 @@
...
+* sunxi: TBD
+* ti-dra7xx: TBD
+* vexpress-fvp: TBD
```
Comment (jenswi-linaro) [inline CHANGELOG.md]: FVP standard test OK

**Comment by jbech-linaro** [comment]
HiKey Debian -> OK. The kernel is a bit dated there, but still seems to work. I've asked 96Boards folks about newer version but I haven't got any reply yet. Since it's working, let's keep the one for this particular release also.
```
23476 subtests of which 0 failed
67 test cases of which 0 failed
0 test case was skipped
TEE test application done!
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,112 @@
...
+* vexpress-juno: TBD
+* vexpress-qemu: TBD
+* vexpress-qemu_armv8a: TBD
```
Comment (jenswi-linaro) [inline CHANGELOG.md]: QEMU v8 standard test OK with https://github.com/OP-TEE/build/pull/76

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,112 @@
...
+* imx-mx6qsabresd: TBD
+* imx-mx6ulevk: TBD
+* ls-ls1021atwr: TBD
+* mediatek-mt8173: TBD
+* rcar-h3: TBD
+* rpi3: TBD
+* sunxi: TBD
+* ti-dra7xx: TBD
+* vexpress-fvp: TBD
+* vexpress-juno: TBD
```
Comment (jenswi-linaro) [inline CHANGELOG.md]: Juno standard test OK with https://github.com/linaro-swg/gen_rootfs/pull/14

**Comment by jforissier** [comment]
Squashed/rebased on top of master, added the Tested-by: tags (I hope I got them right ; maybe next time I'll ask the tester to provide the T-b: in the PR directly). We should be good to go. @jbech-linaro you want to give your R-b maybe?

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by jforissier** [comment]
Added test results for `imx-mx6qsabrelite` and `imx-mx6qsabresd` and merged.

**Pull request merged**